libraries/core/build.gradle.kts

```diff
@@ -35,23 +35,27 @@ dependencies {
     api(project(":libraries:customisations"))
     api(libs.kotlin.coroutines.android)
     api(libs.androidx.lifecycle.common)
-    api(libs.compose.animation.core)
-    api(libs.compose.runtime)
+    api("androidx.compose.animation:animation-core")
```

Comment:
Is this the right way to approach this? I'm not a fan of having two different styles here. Does version catalogs not work with BOMs?

You lose the type safety this way

Reply:
I would suggest looking at this solution: https://stackoverflow.com/questions/67192785/gradle-7-0-version-catalog-for-maven-bom

Reply:
I put them back into catalogue